pkg/bootkube/bootkube.go

```diff
@@ -88,6 +90,10 @@ func makeAPIServerFlags(config Config) []string {
 		"--tls-private-key-file=" + filepath.Join(config.AssetDir, asset.AssetPathAPIServerKey),
 		"--tls-cert-file=" + filepath.Join(config.AssetDir, asset.AssetPathAPIServerCert),
 		"--client-ca-file=" + filepath.Join(config.AssetDir, asset.AssetPathCACert),
+		"--token-auth-file=" + filepath.Join(config.AssetDir, asset.AssetPathBootstrapAuthToken),
+		"--authorization-mode=ABAC,RBAC",
+		"--authorization-policy-file=" + filepath.Join(config.AssetDir, asset.AssetPathAPIServerAuthorizationPolicy),
+		"--authorization-rbac-super-user=admin",
```

Comment:
The "Organization" of a TLS cert can be used to place a client cert in a group btw.

https://kubernetes.io/docs/admin/authentication/#x509-client-certs